packages/query/src/block-processor.ts

```diff
@@ -66,7 +66,7 @@ export class BlockProcessor implements BlockProcessorInterface {
 
   async identifyTransactions(
     spentNullifiers: Set<Nullifier>,
-    commitmentRecords: Map<StateCommitment, SpendableNoteRecord | SwapRecord>,
```

Comment:
There are no meaningful changes in this file — I thought early on in this process that I'd need to modify the block processor to work on this ticket, so I renamed some of the variables to make them a bit clearer. Even after my changes were no longer needed, I figured I'd keep them.